### PR TITLE
Fix linting issues on budget notification implementation

### PR DESCRIPTION
**Pull request by fmasuhr**
The linter is complaining in travis-ci https://travis-ci.org/babbel/terraform-provider-aws/builds/458609620.

As soon as this looks good I will squash this into the original commit for the budget implementation.

### CODE REVIEW

**Comment by fmasuhr** [comment]
[Update our version of the Terraform provider for AWS](https://trello.com/c/7GS1IrpS/1067-update-our-version-of-the-terraform-provider-for-aws)

File: aws/resource_aws_budgets_budget.go [hunk 1]
```diff
@@ -343,6 +343,10 @@ func resourceAwsBudgetsBudgetNotificationRead(d *schema.ResourceData, meta inter
 
 	accountID, budgetName, err := decodeBudgetsBudgetID(d.Id())
 
+	if err != nil {
+		return err
```
Comment (kzonov) [inline aws/resource_aws_budgets_budget.go]: It should probably be smth more meaningful, like here: https://github.com/babbel/terraform-provider-aws/pull/4/files#diff-7b10e1e9bc5b2dcfb62b4a985832c22dR436

Comment (fmasuhr) [reply aws/resource_aws_budgets_budget.go]: @graffzon Yes thanks a lot. I changed it to include some context

**Comment by fmasuhr** [comment]
Merged the changes into the commit for budget notifications manually and pushed to master. This will reduce amount of commits we are ahead of the upstream

**Pull request closed without merging**